### DISCUSSION TITLE
Tagging GHCR Images on the Web

### DISCUSSION THREAD

**Discussion by christianh814** [community/community#135924] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is it possible to tag an image in GHCR "on demand" using the Website? (i.e. **not** using `docker tag` and `docker push` **nor** a GitHub Action)

**Comment by harshsingh-io** [comment]
Currently, tagging an image in the GitHub Container Registry (GHCR) directly through the website without using docker tag and docker push or a GitHub Action is not supported. Tagging typically requires either using Docker commands or automating the process with GitHub Actions.

However, you can manage existing tags and view your images through the GitHub website, but any new tags will need to be created via the command line or through an automated workflow.

If you're looking for a more convenient way to manage tags, consider setting up a simple GitHub Action to handle the tagging process on demand.

**Reply by ebndev** [reply]
Hi @harshsingh-io, thanks for participating in GitHub Discussions!

Please note: we’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  

You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. 

Please feel encouraged to continue engaging in Discussions as your authentic self though. We appreciate you being a part of the Community 🌟
